server/test/controllers/admin/AdminImportControllerTest.java

```diff
@@ -1425,21 +1463,55 @@ public void hxSaveProgram_noDuplicatesEnabled_addsAnEmptyStatus() {
           + "      },\n"
           + "      \"optionalPredicate\" : null,\n"
           + "      \"questionDefinitions\" : [ {\n"
-          + "        \"id\" : 12,\n"
+          + "        \"id\" : 4,\n"
           + "        \"optional\" : false,\n"
           + "        \"addressCorrectionEnabled\" : false\n"
           + "      } ]\n"
+          + "    }, {\n"
+          + "      \"id\" : 3,\n"
+          + "      \"name\" : \"Screen 3\",\n"
+          + "      \"description\" : \"Screen 3 description\",\n"
+          + "      \"localizedName\" : {\n"
+          + "        \"translations\" : {\n"
+          + "          \"en_US\" : \"Screen 3\"\n"
+          + "        },\n"
+          + "        \"isRequired\" : true\n"
+          + "      },\n"
+          + "      \"localizedDescription\" : {\n"
+          + "        \"translations\" : {\n"
+          + "          \"en_US\" : \"Screen 3 description\"\n"
+          + "        },\n"
+          + "        \"isRequired\" : true\n"
+          + "      },\n"
+          + "      \"repeaterId\" : null,\n"
+          + "      \"hidePredicate\" : null,\n"
+          + "      \"eligibilityDefinition\" : {\n"
+          + "        \"predicate\" : {\n"
+          + "          \"rootNode\" : {\n"
+          + "            \"node\" : {\n"
+          + "              \"type\" : \"leafAddressServiceArea\",\n"
+          + "              \"questionId\" : 5,\n"
+          + "              \"serviceAreaId\" : \"Seattle\",\n"
+          + "              \"operator\" : \"IN_SERVICE_AREA\"\n"
```

Comment:
**suggestion:** Here we have a condition for IN_SERVICE_AREA, but it may be worth adding something to validate OUTSIDE_SERVICE_AREA works as well?

Reply:
For reference, it would be `NOT_IN_SERVICE_AREA`